### DISCUSSION TITLE
ScriptReference/Rendering.CommandBuffer.SetGlobalTexture

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#319] [Notes]
# ScriptReference/Rendering.CommandBuffer.SetGlobalTexture

https://docs.unity3d.com/ScriptReference/Rendering.CommandBuffer.SetGlobalTexture.html

**Comment by cjacobwade** [comment]
Sharing a gotcha when using in combination with `CommandBuffer.Blit(source, target, material)`: the global texture set here cannot be the same as the target passed to [`CommandBuffer.Blit`](https://docs.unity3d.com/ScriptReference/Rendering.CommandBuffer.Blit.html) or it will fail to be set
